reports/graphs/feeding_amounts.py

```diff
@@ -14,23 +14,57 @@ def feeding_amounts(instances):
     :param instances: a QuerySet of Feeding instances.
     :returns: a tuple of the the graph's html and javascript.
     """
-    totals = {}
+    feeding_types = [
+            'breast milk',
+            'formula',
+            'fortified breast milk',
+            'solid food',
+        ]
+    feeding_types_desc = [
+            'Breast milk',
+            'Formula',
+            'Fortified breast milk',
+            'Solid food',
+        ]
+    total_idx = len(feeding_types) + 1  # +1 for aggregate total
+    totals_list = list()
+    for i in range(total_idx):
+        totals_list.append({})
     for instance in instances:
         end = timezone.localtime(instance.end)
         date = end.date()
-        if date not in totals.keys():
-            totals[date] = 0
-        totals[date] += instance.amount or 0
-
-    amounts = [round(amount, 2) for amount in totals.values()]
-    trace = go.Bar(
-        name=_('Total feeding amount'),
-        x=list(totals.keys()),
-        y=amounts,
+        if date not in totals_list[total_idx-1].keys():
+            for item in totals_list:
+                item[date] = 0
+        feeding_idx = feeding_types.index(instance.type)
+        totals_list[feeding_idx][date] += instance.amount or 0
+        totals_list[total_idx-1][date] += instance.amount or 0
+    zeros = [0 for a in totals_list[total_idx-1].values()]
+
+    # sum each feeding type for graph
+    amounts_array = []
+    for i in range(total_idx):
+        amounts_array.append([round(a, 2) for a in totals_list[i].values()])
+
+    traces = []
+    for i in range(total_idx-1):
+        traces.append(go.Bar(
+            name=feeding_types_desc[i],
+            x=list(totals_list[total_idx-1].keys()),
+            y=amounts_array[i],
+            text=amounts_array[i]
```

Comment:
On hover right now we get too much information -- date, amount again, and type. Ideally we just have type.

<img width="235" alt="image" src="https://user-images.githubusercontent.com/10456740/152088539-ad764ebc-a431-441e-88b3-9cf1c6f801d7.png">

I'm not 100% sure how to set it up that way -- take a look at https://plotly.com/python/hover-text-and-formatting/ and if ya get stuck I can take a closer look.